inst/components/scss/sidebar.scss

```diff
@@ -235,90 +267,80 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
 }
 
 @include media-breakpoint-down(sm) {
-  .bslib-sidebar-layout, .bslib-sidebar-layout.sidebar-right {
-    --bslib-sidebar-vert-border: none;
-    --bslib-sidebar-horiz-border: #{$bslib-sidebar-border};
-    --bslib-collapse-toggle-transform: -180deg;
-    --bslib-collapse-toggle-right-transform: -180deg;
-
-    // required by sidebar init js when `sidebar(open = "desktop")`
-    &[data-bslib-sidebar-open="desktop"] {
-      --bslib-sidebar-js-init-collapsed: true;
+  .bslib-sidebar-layout {
+    // Tell sidebar init js we're on mobile for `sidebar(open = "desktop")`
+    &, &.sidebar-right {
+      &[data-bslib-sidebar-open="desktop"] {
+        --bslib-sidebar-js-init-collapsed: true;
+      }
     }
 
-    grid-template-columns: 1fr !important;
-    // Sidebar height is constrained on mobile where upper bound is determined
-    // by bslib UI functions. The minimax() is important to ensure the main main
-    // content is allowed to grow/shrink.
-    grid-template-rows:
-      fit-content(var(--bslib-sidebar-max-height-mobile, auto))
-      minmax(0, 1fr);
-
-    > .sidebar {
-      grid-row: 1 / 2;
-      grid-column: 1 / 2;
-      width: 100%;
-      border: none;
-      border-bottom: var(--bslib-sidebar-horiz-border);
-      border-radius: 0;
+    // Remove left or right edge sidebar borders in mobile view
+    &:not(.sidebar-right) {
+      > .sidebar { border-right: none }
     }
-
-    > .main {
-      grid-row: 2 / 3;
-      grid-column: 1 / 2;
-      border-top-left-radius: 0;
-      border-top-right-radius: 0;
-      border-bottom-right-radius: inherit;
-      border-bottom-left-radius: inherit;
+    &.sidebar-right {
+      > .sidebar { border-left: none }
     }
```

Comment:
Could all of this be just `.sidebar{border:none}`? (but we might need `border-bottom` for the alywas open?)

Reply:
Yes, done in 020eb11e906602ab3145221c85122397a4344c77 and I added the border-top for always open sidebars